src/test/java/com/stripe/model/EventTest.java

```diff
@@ -31,11 +31,11 @@ public void serializesToJson() throws IOException {
 
 		assertEquals(reserializedEvent.getId(), event.getId());
 		assertEquals(reserializedEvent.getObject(), event.getObject());
+		assertEquals(reserializedEvent.getAccount(), event.getAccount());
```

Comment:
I fully admit that it kind of sucks, but it works because both values are `null`.